utils/frame/benchmarking-cli/src/pallet/mod.rs

```diff
@@ -131,6 +134,17 @@ pub struct PalletCmd {
 	)]
 	pub wasm_method: WasmExecutionMethod,
 
+	/// The WASM instantiation method to use.
+	///
+	/// Only has an effect when `wasm-execution` is set to `compiled`.
+	#[clap(
+		long = "wasm-instantiation-strategy",
+		value_name = "STRATEGY",
+		default_value_t = DEFAULT_WASM_INSTANTIATION_STRATEGY,
+		arg_enum,
+	)]
+	pub wasm_instantiation_strategy: WasmInstantiationStrategy,
```

Comment:
Hmm... I don't really have a strong opinion about it, but it does feel a little strange to have a `compiled` execution method (which doesn't say anything about `wasmtime`) and then have another `wasmtime-` argument, and it's not immediately obvious that the two are related. (But I guess you could say the same thing here, that it's not obvious that this only applies to the compiled execution.)

How about we do this?

1. Rename `compiled` to `wasmtime`.
2. For backwards compatibility still accept `compiled` and have it mean `wasmtime` by default. (And maybe print a warning?)
3. Rename this as you're suggesting to `wasmtime-`.

Reply:
Yes, this is roughly what I was thinking about. 

I did not think we should do the whole thing now. This would just inflate the PR.  Although doing the number 3 now would be better right away IMO so that we don't release it and then change it.

Reply:
Sounds good to me; I'll do 1 and 2 in a followup PR.